actions/bumper/entrypoint.sh

```diff
@@ -51,15 +51,15 @@ git fetch --tags
 
 # get latest tag & version that looks like a semver (with or without v, using with_v)
 if $with_v; then
-    tag_pattern="refs/tags/v[0-9]*.[0-9]*.[0-9]*"
+    tag_pattern="v[0-9]*.[0-9]*.[0-9]*"
     version_pattern="(.*)v([0-9]+\.[0-9]+\.[0-9]+-?[a-zA-Z0-9]*)(.*)"
 else
-    tag_pattern="refs/tags/[0-9]*.[0-9]*.[0-9]*"
+    tag_pattern="[0-9]*.[0-9]*.[0-9]*"
     version_pattern="(.*)([0-9]+\.[0-9]+\.[0-9]+-?[a-zA-Z0-9]*)(.*)"
 fi
 
 case "$tag_context" in
-    *repo*) tag=$(git for-each-ref --sort=-v:refname --count=1 --format '%(refname)' "$tag_pattern" | cut -d / -f 3-);;
+    *repo*) tag=$(git for-each-ref --sort=-committerdate --count=1 --format '%(refname)' "refs/tags/$tag_pattern" | cut -d / -f 3-);;
```

Comment:
Sorting by `refname` gives alphabetical order but `v:refname` should [treat these as versions and sort appropriately](https://git-scm.com/docs/git-for-each-ref). Are you seeing different behavior?

In general I also don't think we should always assume the last created tag is the latest, which won't work for any libraries maintaining two+ major versions (though I don't think that's currently an issue)

Reply:
I found few errors while testing manual tags and release logic locally. 
Yes, they will be treated as versions (corrected from: plain alphabetic order), but an out of order manual tag / version can break this. 

going by ref-name also does not mean the ref sorted alphabetically is not the latest. 
Since the tags dont have a 'latest' annotation, the only way to identify the latest tag is going by the date
Additionally, both actions (action-releaser and bumper) should use the same logic to identify the latest tag

multiple major / minor versions: This case is not supported in the current implementation either. 
Version is maintained in the version file, and a single version along is supported. 

Releases do have a 'latest' annotation, but releases are directly linked to specific tags